dbrp/service.go

```diff
@@ -394,6 +387,15 @@ func (s *Service) Create(ctx context.Context, dbrp *influxdb.DBRPMappingV2) erro
 		if !defSet {
 			dbrp.Default = true
 		}
+
+		b, err := json.Marshal(dbrp)
+		if err != nil {
+			return ErrInternalService(err)
+		}
```

Comment:
There is a slight benefit to marshaling outside the `Update`, as we reduce the time we hold a lock. It probably isn't a significant deal, but something to consider. Presumably it is unlikely to fail, but if it does, we save taking a lock in that instance too.

Reply:
Good call. I'll revisit.

Reply:
Going to go ahead and merge this, because I think the risk of having problems is low here. Especially since we aren't hitting cache with this operation.